releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,17 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 7.0.0-preview - July 20, 2023
+### Breaking Changes:
+* Feature flags are now parsed and transformed into feature management configuration even if `AzureAppConfigurationOptions.UseFeatureFlags` is not called. `AzureAppConfigurationOptions.UseFeatureFlags` is still used to configure feature flag options. [#422](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/422)
+
+### Enhancements
+* Added the following new API for adding a snapshot's key-values to the configuration. [#422](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/422)
```

Comment:
@jimmyca15 do we have any public doc for snapshots yet? It might be good to add a link here. What do you think?

Reply:
It's a great idea. Unfortunately, at the moment we only have [this low-level doc](https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-snapshot) talking about the REST API. 

[This PR](url) is in the works to add concept docs. Once this one is done, this is the doc that will be great to link to.

Reply:
Doc PR link is broken, but I get your point. We can add it in a follow up PR once the public doc is live.

Reply:
Link fixed.